src/Stripe.net.Tests/Stripe.net.Tests.csproj

```diff
@@ -31,22 +52,4 @@
     <EmbeddedResource Include="infrastructure\test_data\subscription.json" />
   </ItemGroup>
 
```

Comment:
My gut reaction is this is the issue. If it's not targetting netstandard, it will need to pull in the assemblies from the .net framework it needs to run. I'll check this out - I don't think we need all of these anyway.